test/baddtlstest.c

```diff
@@ -0,0 +1,617 @@
+/*
```

Comment:
Completely drive-by, but please use underscores to separate words in the filename. (Not all OpenSSL C sources follow this, but a majority do.) This is `bad_dtls_test` not `badd_tls_test` :)